dashboard/app/models/script.rb

```diff
@@ -459,6 +460,35 @@ def self.add_script(options, data)
 
     script.stages = script_stages
     script.reload.stages
+
+    # Generate PLC objects
+    if script.professional_learning_course
+      course = Plc::Course.find_or_create_by! name: 'Default'
```

Comment:
I'd like to build out the concept of a `Progression`/`Sequence` in a way that will apply to `Script` as well.  Until then I've create a "Default" course to link the CSP content to.  If you prefer I can call it "Computer Science Principles"?